src/compas/geometry/point.py

```diff
@@ -130,7 +130,7 @@ def __str__(self):
             self.x,
             self.y,
             self.z,
-            TOL.PRECISION,
+            TOL.precision,
```

Comment:
technically, precision can actually also be on the other side of the decimal point. so, technically, better would be
```python
"{0}(x={1}, y={2}, z={3})".format(TOL.format_number(self.x), TOL.format_number(self.y), TOL.format_number(self.z))
```
but this also works

Reply:
:)

Reply:
Ah! True! Fixed at https://github.com/compas-dev/compas/pull/1234/commits/dc19b7d42707e24b44ac0b773fa3020f5d6ca8ab